jdaviz/configs/mosviz/helper.py

```diff
@@ -211,8 +211,6 @@ def load_data(self, spectra_1d=None, spectra_2d=None, images=None,
             ``images``. Can be a list of strings representing data labels
             for each item in ``data_obj`` if  ``data_obj`` is a list.
         """
-        # Link data after everything is loaded
-        self.app.auto_link = False
```

Comment:
And I don't understand why you are removing auto link stuff.

Reply:
Ditto.